sp-sim/examples/config.toml

```diff
@@ -3,19 +3,39 @@
 #
 
 [[simulated_sps.sidecar]]
-multicast_addr = "ff15:0:1de::0"
-bind_addrs = ["[::]:33300", "[::]:33301"]
 serial_number = "SimSidecar0"
 manufacturing_root_cert_seed = "01de01de01de01de01de01de01de01de01de01de01de01de01de01de01de01de"
 device_id_cert_seed = "01de000000000000000000000000000000000000000000000000000000000000"
 
+[[simulated_sps.sidecar.network_config]]
+[simulated_sps.sidecar.network_config.Real]
+bind_addr = "[::1]:33300"
+multicast_addr = "ff15:0:1de::0"
+multicast_interface = "net0"
```

Comment:
It seems quite unfortunate that this file no longer works without replacing `net0` / `net1` with real interface names on whatever system this is being run. I'm not sure what to suggest; if we have to have this, we at least need to update the wicket README to tell people to edit these values before running sp-sim (but to what?).

Could we make `multicast_interface` optional and continue to pass index 0 to `join_multicast_v6()` if it isn't provided? Or if that is nonsensical, can we change this config to use simulated instead of real?

Reply:
I've reverted this to the 'simulated' config in f0d59b75aed49f7cb1e4f6db744ca0314cdd3538, which should match the previous behaviour (except for the incorrect multicast config) and avoid the need to match interface names. Thanks for the catch :)